### PR TITLE
Feature/#8065 Get campaign submissions endpoint 

### PR DESCRIPTION
**Pull request by James-Thorp**
Connects https://github.com/CorporateRewards/myrewards/issues/8065

### CODE REVIEW

File: source/includes/api/recognition/_submissions.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+#### Query Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+page | `integer` | The page number that you wish to view. Optional
+status | `string` | Status of the submissions to be returned. Optional
+user_id | `integer` | ID of a sender that submissions were made by. Optional
+created_at_date | `string` | A parameter for scoping submissions that were created from that date onwards. Optional.
+created_at_end_date | `string` | A parameter for scoping submissions that were created on or before the given date. Optional.
```
Comment (rjpaskin) [inline source/includes/api/recognition/_submissions.md:19]: Should this be `date`, not `string`?

Comment (rjpaskin) [reply source/includes/api/recognition/_submissions.md:19]: Also what's the date format expected?

File: source/includes/api/recognition/_submissions.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        },
+        "points_approved": 100,
+        "approved_by": {
+            "level_1_id": 505,
+            "level_2_id": null
+        },
+        "declined_by": {
+            "level_1": {
+                "id": null,
```
Comment (rjpaskin) [inline source/includes/api/recognition/_submissions.md:65]: What model does `id` refer to here? If it's user, could we use `user_id` instead for clarity?

File: source/includes/api/recognition/_submissions.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            },
+            "1176": {
+                "question": "Custom q 3 - lov c",
+                "answer": ""
+            }
+        },
+        "points_approved": 100,
+        "approved_by": {
+            "level_1_id": 505,
+            "level_2_id": null
```
Comment (rjpaskin) [inline source/includes/api/recognition/_submissions.md:61]: Match structure of `declined_by`?
```suggestion
            "level_1": { "id": 505 },
            "level_2": { "id": null }
```
`id` here would be `user_id` if that's what the `id` refers to

**Review by rjpaskin — CHANGES_REQUESTED** [review]

File: source/includes/api/recognition/_submissions.md (hunk 1 above)
Comment (James-Thorp) [reply source/includes/api/recognition/_submissions.md:19]: I used `string` to keep it consistent with how we have it elsewhere in the docs (i.e. in self claims) but I can change it to date 👍
The expected format is ISO-8601. I'll add that in, thanks.

File: source/includes/api/recognition/_submissions.md (hunk 2 above)
Comment (James-Thorp) [reply source/includes/api/recognition/_submissions.md:65]: Yes good suggestion, I'll amend it

**Pull request merged**